src/c++/perf_analyzer/client_backend/mock_client_backend.h

```diff
@@ -124,6 +124,35 @@ class MockClientStats {
     std::atomic<size_t> num_close_shared_memory_calls{0};
     std::atomic<size_t> num_unlink_shared_memory_region_calls{0};
     std::atomic<size_t> num_unmap_shared_memory_calls{0};
+
+    // bool operator==(const SharedMemoryStats& lhs, const SharedMemoryStats&
+    // rhs)
+    bool operator==(const SharedMemoryStats& rhs) const
```

Comment:
Is there no default operator== for structs? Does this skip some data members that don't require equality across lhs and rhs in order for lhs and rhs to be considered "equal"?

Reply:
Not until c++20, where the compiler can infer one. Until then, we have to write one